truckersmp_cli/utils.py

```diff
@@ -353,6 +353,42 @@ def get_short_size(size_bytes):
     return "{:.1f}M".format(size_bytes / 1048576)
 
 
+def get_steam_library_dirs(steamdir):
+    """
+    Get Steam library directories.
+
+    steamdir: Steam installation path
+    """
+    # the first Steam library directory is the Steam installation
+    steam_libraries = [steamdir, ]
```

Comment:
Isn't this ?
```suggestion
    # the first Steam library directory is the Steam installation
    steam_libraries = [os.path.join(steamdir, "steamapps"), ]
```

Beware older installations may still have `os.path.join(steamdir, "SteamApps")`